tests/kwargs/test_path_params.py

```diff
@@ -127,9 +127,9 @@ def test_method() -> None:
         ["float", float, 1.01],
         ["uuid", UUID, uuid4()],
         ["decimal", Decimal, Decimal("1.00001")],
-        ["date", date, date.today().isoformat()],
-        ["datetime", datetime, datetime.now().isoformat()],
-        ["timedelta", timedelta, timedelta(days=1).total_seconds()],
+        # ["date", date, date.today().isoformat()],
+        # ["datetime", datetime, datetime.now().isoformat()],
+        # ["timedelta", timedelta, timedelta(days=1).total_seconds()],
         ["path", Path, "/1/2/3/4/some-file.txt"],
```

Comment:
I think we should discuss whether we need `date`, `datetime` and `timedelta` as path parameter types.

As we say in the docs:
> ...as long as parameter inside the function declaration is typed with a "higher" type to which the lower type can be coerced, this is fine

Whenever these are used the type is declared in the handler/dependency signature and pydantic does the parsing anyway.

We could avoid making the change breaking by aliasing them to `str`.